packages/amplify-cli/src/extensions/amplify-helpers/show-auth-acm.ts

```diff
@@ -27,65 +31,71 @@ export function showACM(sdl: string, nodeName: string) {
       const authRules: AuthRule[] = getAuthDirectiveRules(new DirectiveWrapper(parentAuthDirective));
       convertModelRulesToRoles(acm, authRules);
     }
-    for (let fieldNode of type.fields || []) {
-      let fieldAuthDir = fieldNode.directives?.find(dir => dir.name.value === 'auth') as DirectiveNode;
+
+    (type.fields || []).forEach(fieldNode => {
+      const fieldAuthDir = fieldNode.directives?.find(dir => dir.name.value === 'auth') as DirectiveNode;
       if (fieldAuthDir) {
         if (parentAuthDirective) {
           acm.resetAccessForResource(fieldNode.name.value);
         }
         const authRules: AuthRule[] = getAuthDirectiveRules(new DirectiveWrapper(fieldAuthDir));
         convertModelRulesToRoles(acm, authRules, fieldNode.name.value);
       }
-    }
+    });
     const truthTable = acm.getAcmPerRole();
 
     if (truthTable.size === 0) {
       printer.warn(`No auth rules have been configured for the "${type.name.value}" model.`);
     }
 
-    for (let [role, acm] of truthTable) {
+    truthTable.forEach((table, role) => {
```

Comment:
Double check if this should be `truthTable.forEach((role, table) =>`

Reply:
no, for/of map iterator is `[key, value]`, while forEach is `(value, key)`